processing/data_collection/gazette/spiders/pr_foz_do_iguacu.py

```diff
@@ -0,0 +1,71 @@
+import datetime as dt
+
+import scrapy
+from scrapy.http import FormRequest
+
+from gazette.items import Gazette
+
+
+class PrFozDoIguacuSpider(scrapy.Spider):
+    MUNICIPALITY_ID = '4108304'
+    name = 'pr_foz_do_iguacu'
+    allowed_domains = ['pmfi.pr.gov.br']
+    start_urls = ['http://www.pmfi.pr.gov.br/utilidades/diario/index.xhtml']
+    AVAILABLE_FROM = dt.date(2015, 1, 1)  # actually September/2007
+
+    def parse(self, response):
```

Comment:
You could implement `start_requests` method instead of putting this in the `parse` method.
More information: https://github.com/okfn-brasil/diario-oficial/pull/40#issuecomment-389587364

Reply:
@rennerocha, I have used the parse because I need the number of documents available. Do you see a way of doing that using the start_requests?